service/products.ts

```diff
@@ -0,0 +1,71 @@
+import { zeppelinsDb } from '~/api/zeppelinsDb';
+import { Product } from '~/types/products';
+
+export const getProductList = async (count?: number) => {
+  try {
+    if (count) {
+      const { data } = await zeppelinsDb
+        .from('products')
+        .select('*, product_specifications (range)')
+        .limit(count);
+      return data;
```

Comment:
I wonder if it wouldn't be better to return everything that supabase returns? As there is some more stuff than only data and it may be useful to have it there (even if now we don't need it).

Also what if `const { data, error } = await zeppelinsDb.from()...` returns error there, does it "throw" and will be caught by `catch` block?

Reply:
I will have a look. Not checked yet what whole return object is.

`const { data, error }` - Still not forced to error but from syntax my understanding is that it would not throw an error by default, we would have to handle this on FE

Reply:
There is actually not more there. It's an array that consists of array of data and i presume another one for error when one occurs.